clover_admin.rb

```diff
@@ -220,6 +221,23 @@ def self.object_action(...)
       end
     end
 
+    model Vm do
+      order Sequel.desc(:created_at)
+      eager [:location, :vm_host]
+      columns [:name, :display_state, :vm_host, :location, :arch, :boot_image, :family, :vcpus, :created_at]
+      column_options display_state: {type: "select", options: ["running", "creating", "starting", "rebooting", "deleting"], add_blank: true},
```

Comment:
I think this needs special support when searching, since `display_state` is not a database column.

Reply:
It's a column for VM model

Reply:
Previously, I tried to refactor it to be similar to other models, but it required more work than I expected.